dataInterface/items.js

```diff
@@ -2,22 +2,26 @@ const uuid = require('uuid');
 
 module.exports = {};
 
-module.exports.items = [{ id: '7', field: 'example' }];
+module.exports.items = [{ id: '7', field: 'example' }, {id:'test1', field: 'example1'}, {id:'test2', field: 'example2'}];
 
 module.exports.getAll = () => {
   return module.exports.items;
 }
 
 module.exports.getById = (itemId) => {
-  // TODO: complete writing this function
+  return module.exports.items.find(ele => ele.id === itemId);
 }
 
 module.exports.deleteById = async (itemId) => {
-    // TODO: complete writing this function
+    const index = module.exports.items.findIndex(ele => ele.id === itemId)
+    return module.exports.items.splice(index, 1);
 }
 
 module.exports.updateById = async (itemId, newObj) => {
-    // TODO: complete writing this function
+   const element = module.exports.items.find(ele => ele.id === itemId)
+   if(element){
+    element.field = newObj.field;
+   }
 }
```

Comment:
If the return value of this function is returned to the caller from the route handler, shouldn't this return something?

Reply:
yes, it should return the element object.